images/macos/toolsets/toolset-11.0.json

```diff
@@ -83,7 +83,11 @@
         "addon-list": [],
         "additional-tools": [
             "cmdline-tools;latest"
-        ]
+        ],
+        "ndk": {
+            "lts": "21.3.6528147",
+            "latest": "22.0.7026061"
```

Comment:
Can we specify only the major version here and install the latest minor for it? Because lts and latest can be updated through the year.

Reply:
Done.